src/visitors/ispc_rename_visitor.cpp

```diff
@@ -0,0 +1,64 @@
+/*************************************************************************
+ * Copyright (C) 2018-2020 Blue Brain Project
+ *
+ * This file is part of NMODL distributed under the terms of the GNU
+ * Lesser General Public License. See top-level LICENSE file for details.
+ *************************************************************************/
+
+#include "visitors/ispc_rename_visitor.hpp"
+
+#include "ast/all.hpp"
+#include "parser/c11_driver.hpp"
+#include "visitors/visitor_utils.hpp"
+
+
+namespace nmodl {
+namespace visitor {
+
+/// rename matching variable
+void IspcRenameVisitor::visit_name(ast::Name& node) {
+    const auto& name = node.get_node_name();
+    if (std::regex_match(name, double_regex)) {
+        auto& value = node.get_value();
+        const auto& vars = get_global_vars(*ast);
+        value->set(suffix_random_string(vars, new_var_name_prefix + name));
```

Comment:
We should show warning from logger here because user might be accessing these variables from neuron and renaming those would be a problem. With warning, at least they will be aware (if they read!) .

Reply:
you know.. when someone does some `VERBATIM` ... 😄

Reply:
also do we need the random suffix here? the names are already unique, just not ispc compatible, the prefix should already take care of that.

Reply:
@pramodk you're absolutely right about the log for the renaming
@ohm314 I added the random suffix just in case there is already a `var_d1` variable already defined. There is a bug the way I do this now though I think which I need to fix

Reply:
Yes of course! actually now I remember when I did the renaming stuff I did the same, sorry!